app/views/govuk_component/title.raw.html.erb

```diff
@@ -2,8 +2,12 @@
   average_title_length ||= false
   context ||= false
   inverse ||= false
+  margin_bottom ||= 1
+  margin_bottom_class = "pub-c-title--bottom-margin" if margin_bottom == 1
```

Comment:
Maybe:
```
margin_bottom ||= true
margin_bottom_class = "pub-c-title--bottom-margin" if margin_bottom
```

Reply:
Don't think that works - if you pass 0 to the component that counts as true.

Reply:
```
margin_bottom_class = "pub-c-title--bottom-margin" if margin_bottom !== 0
```

?

Reply:
That doesn't work either... margin_bottom is not defined.